telemetry-sensing/telemetry-sensing/lib/TEMPERATURE/TEMPERATURE.cpp

```diff
@@ -0,0 +1,46 @@
+#include <TEMPERATURE.h>
+
+namespace TEMPERATURE{
+  void init(){
+    pinMode(TEMP_PIN_0, INPUT); 
+    pinMode(TEMP_PIN_1, INPUT); 
+    pinMode(TEMP_PIN_2, INPUT);
+    pinMode(TEMP_PIN_3, INPUT);
+    pinMode(TEMP_PIN_4, INPUT); 
+    pinMode(TEMP_PIN_5, INPUT);  
+  }
+  
+  float floatMap(float x, float in_min, float in_max, float out_min, float out_max) {
+    return (x - in_min) * (out_max - out_min) / (in_max - in_min) + out_min;
+  }
+
+  float getVoltage(int16_t pin) {
+    return Voltage - floatMap(analogRead(pin), 0, 1023, 0.000, Voltage);
+  }
+
+  float voltageToResistance(int16_t pin) {

```

Comment:
Since input is a pin this should probably just be called getResistance

Reply:
I think is oki :)